ush/hpssarch_gen.sh

```diff
@@ -9,6 +9,8 @@ set -x
 
 type=${1:-gfs}                ##gfs, gdas, enkfgdas or enkfggfs
 
+COMPONENTatmos=${COMPONENTatmos:-atmos}
```

Comment:
I thought the decision was not to use COMPONENTatmos and COMPONENTwave

Reply:
You're right, we should use COMINatmos and COMINwave instead. Those variables are both overkill and hard to use in this script since multiple dirpaths get set, so I will change it back to the hardcoded values of "atmos" and "wave".

Reply:
Changed it back to the hardcoded settings.